src/nvim/vvars.lua

```diff
@@ -771,12 +771,20 @@ M.vars = {
   },
   termresponse = {
     desc = [=[
-      The value of the most recent OSC or DCS escape sequence
+      The value of the most recent OSC or DCS control sequence
       received by Nvim from the terminal. This can be read in a
       |TermResponse| event handler after querying the terminal using
       another escape sequence.
     ]=],
   },
+  termrequest = {
```

Comment:
Please consider adding a type (maybe v:termresponse could also get one).

```suggestion
  termrequest = {
    type = "string",
```